rct229/rulesets/ashrae9012019/section5/section5rule1.py

```diff
@@ -112,7 +112,7 @@ def create_data(self, context, data):
 
             # filter out baseline rmds that aren't None then count the length
             no_of_output_instance = len(
-                [rmd_b["type"] for rmd_b in baseline_rmd_list if rmd_b is not None]
+                [rmd["type"] for rmd in rmd_list_no_user if rmd is not None]
```

Comment:
Shouldn't this count the number of `output_instances` in the rmd files?
```python
len(compact([find_one($.output.output_instance) for rmd in rmd_list_no_user]))
```

Reply:
Great. Thank you Weili!